### DISCUSSION TITLE
Problem installing on macOS Catalina 10.15.6

### DISCUSSION THREAD

**Discussion by cod3licious** [qpdf/qpdf#510] [Q&A]
I tried to install qpdf-10.0.1 from sourceforge on my mac, but when running `.configure` I got the warning/error
```
configure: WARNING: unable to find required header jpeglib.h
configure: WARNING: unable to find required library jpeg

configure: error: some required prerequisites were not found
```
I then installed the jpeg library from https://libjpeg-turbo.org/ as mentioned in the manual, which seems to have been installed in `/opt/libjpeg-turbo/` but the error persists... Is there any way to fix this?

**Comment by ghost** [comment]
I've had the jpeg from http://www.ijg.org/files/ installed. It managed to get rid of the second WARNING but the first remained there. Trouble was that I could easily find "jpeglib.h" under \usr\local\include. There is something amiss with the CONFIGURE script. Manually setting the environment variable by running "./configure CPPFLAGS=-I/usr/local/include" got rid of the issue.

QPDF 10.2.0 up and running on macOS 11.2.1

**Reply by silicontrip** [reply]
You know that brew has version 10.1.0 of qpdf. I'd highly look at using a package manager, as others have done the hard work of figuring out how to configure and build it.
